api/src/pcapi/core/subscription/api.py

```diff
@@ -51,37 +51,40 @@ def activate_beneficiary_if_no_missing_step(user: users_models.User) -> bool:
             .with_for_update()
             .one()
         )
-        subscription_state = get_user_subscription_state(user)
 
-        if not subscription_state.is_activable:
-            return False
-        if not subscription_state.identity_fraud_check:
-            return False
-        if subscription_state.identity_fraud_check.resultContent is None:
+        subscription_state_machine = subscription_machines.create_state_machine_to_current_state(user)
+        if (
+            not subscription_state_machine.state
+            == subscription_machines.SubscriptionStates.SUBSCRIPTION_COMPLETED_BUT_NOT_BENEFICIARY_YET
```

Comment:
!=

Reply:
```suggestion
            != subscription_machines.SubscriptionStates.SUBSCRIPTION_COMPLETED_BUT_NOT_BENEFICIARY_YET
```
Pardon, je voulais juste dire qu'utiliser `!=` au lieu de `not ... ==` était plus idiomatique.
Le nom est très clair.